vignettes/handleiding_trendanalyse.Rmd

```diff
@@ -289,31 +299,38 @@ ggplot(a, aes(x = n, fill = above)) +
 
 ```
 
-  b) *verdeling waarnemingen 1998-2018* van de reeksen van waarnemingen
-  per stof per putfilter dienen ten minste twee waarnemingen in de 
+  b) *verdeling waarnemingen 1998-2024* van de reeksen van waarnemingen
+  per stof per putfilter dienen ten minste een waarneming in de 
   eerste planperiode KRW te zijn gelegen (2004-2009), ten minste twee
   waarnemingen in de tweede planperiode KRW (2010-2015) en ten minste
-  één waarneming in de derde planperiode KRW (2016-2021). Markeer de 
-  reeks indien hier niet aan wordt voldaan. 
+  twee waarneming in de derde planperiode KRW (2016-2024). Als een reeks niet 
+  voldoet aan dit criterium, dient de reeks te worden afgekeurd. 
   
   Een overzicht per put kan verkregen worden met de `meetverdeling()`
   functie. Reeksen met onvoldoende verdeling in waarnemingen kunnen
-  vervolgens worden gemarkeerd. Zie onderstaand script voor een voorbeeld:
+  vervolgens worden afgekeurd. Zie onderstaand script voor een voorbeeld:
 
 ```{r}
+# Standaard periodes
 d <- grondwaterdata %>%
   filter(parameter == "no3", diepte == "ondiep", putfilter == "5f1") 
 
 resultaat <- meetverdeling(d)
 
 knitr::kable(resultaat)
 
+# Periodes met overlap
```

Comment:
Dit hele concept 'overlap' is mij niet duidelijk uit de tekst. Wat is het? Wanneer gebruik je het? En hoe bepaal hoeveel periodes je nodig hebt?